google/cloud/storage/version.h

```diff
@@ -64,10 +62,19 @@ int constexpr version_minor() { return google::cloud::version_minor(); }
  */
 int constexpr version_patch() { return google::cloud::version_patch(); }
 
+/**
+ * Returns the Google Cloud Storage C++ Client pre-release version.
+ *
+ * @see https://semver.org/spec/v2.0.0.html for details.
+ */
+constexpr char const* version_pre_release() {
```

Comment:
Is that the `east const` form?

Reply:
It's the "east const but constexpr west" form.  https://youtu.be/z6s6bacI424